### DISCUSSION TITLE
vue-devtools使用无效

### DISCUSSION THREAD

**Discussion by syifeng** [vbenjs/vue-vben-admin#449] [Q&A]
请教一下，在使用vue-vben-admin时不论是chrome还是Firefox新版的还是旧版的vue-devtools都无法使用

**Comment by syifeng** [comment]
新版和旧版的vue-devtools都安装了，按照搜到的方法，禁用了老板的工具，就可以了

**Reply by IFnGSiYu** [reply]
新版应该是兼容vue2的
